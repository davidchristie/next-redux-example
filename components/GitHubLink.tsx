import React from "react";

const GitHubLink: React.FunctionComponent = () => (
  <a href="https://github.com/davidchristie/next-redux-example">GitHub</a>
);

GitHubLink.displayName = "GitHubLink";

export default GitHubLink;
