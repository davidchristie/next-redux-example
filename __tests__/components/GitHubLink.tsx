import GitHubLink from "../../components/GitHubLink";
import {
  describeComponent,
  itHasLink,
  itShouldMatchSnapshot
} from "../../test-utils";

describeComponent(GitHubLink, {}, getContext => {
  itHasLink(
    getContext,
    "GitHub",
    "https://github.com/davidchristie/next-redux-example"
  );

  itShouldMatchSnapshot(getContext);
});
