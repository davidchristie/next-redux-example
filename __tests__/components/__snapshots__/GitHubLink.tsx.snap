// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubLink Component Should match snapshot 1`] = `
<DocumentFragment>
  <a
    href="https://github.com/davidchristie/next-redux-example"
  >
    GitHub
  </a>
</DocumentFragment>
`;
